script.js

```diff
@@ -6,3 +6,12 @@ function openNav() {
 function closeNav() {
   document.getElementById("sidebar").style.width = "0";
 }
+
+function toastToggle(id) {
```

Comment:
add some timeout to the toast, so that after a certain time the toast disappears.

Reply:
Done the Changes

Reply:
use es6

Reply:
Done the Changes